coordinator/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gin-gonic/gin v1.9.1
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/scroll-tech/go-ethereum v1.10.14-0.20231130005111-38a3a9c9198c
+	github.com/scroll-tech/go-ethereum v1.10.14-0.20240201173512-ae7cbae19c84
```

Comment:
Shall we use the latest one? https://github.com/scroll-tech/go-ethereum/commit/a807a41fd1e9d1b7ec7d371ef1188fa6be209faa

Reply:
update [009fc9b](https://github.com/scroll-tech/scroll/commit/009fc9b0aec7738db6e072808c519ef2f02eed10)